components/prism-cmake.js

```diff
@@ -1,8 +1,18 @@
-Prism.languages.cmake = Prism.languages.extend('python', {
+Prism.languages.cmake = {
+    'comment': {
+		pattern: /(^|[^\\])#.*/,
+		lookbehind: true
+    },
+    'string': {
+		pattern: /(?:[rub]|rb|br)?("|')(?:\\.|(?!\1)[^\\\r\n])*\1/i,
```

Comment:
Now that we control strings, we might as well remove the `(?:[rub]|rb|br)?` prefix (and the `i` flag).

I also have a few questions about strings:

1. Do backslash escapes work? (e.g. `"\n\t"`)
2. Are multiline strings allowed?

Reply:
Multiline strings - yes.
Bashslash escapes - no.

Reply:
Simplified.

Reply:
Alright, then this should work:

```js
'string': {
    pattern: /"(?:[^\\"]|\\.)*"/,
    greedy: true
}
```

Reply:
Fixed